src/menu.js

```diff
@@ -0,0 +1,45 @@
+function Menu() {
+  const menuDiv = document.createElement('div');
+  menuDiv.setAttribute('class', 'flex-column');
+
+  const starters = document.createElement('div');
+  starters.setAttribute('class', 'flex-display pb-10');
+
+  const fishes = document.createElement('div');
+  fishes.setAttribute('class', 'flex-display pb-10');
+
+  const salads = document.createElement('div');
+  salads.setAttribute('class', 'flex-display pb-10');
+
+  const desserts = document.createElement('div');
+  desserts.setAttribute('class', 'flex-display pb-10');
+
+  const spaceI = document.createElement('div');
+  spaceI.setAttribute('class', 'space');
+
+  const spaceII = document.createElement('div');
+  spaceII.setAttribute('class', 'space');
+
+  const spaceIII = document.createElement('div');
+  spaceIII.setAttribute('class', 'space');
```

Comment:
This code is redundant: you're using the same element in 3 different variables.
Also, you do not need to use empty `div` (filled with the `<br>` tags) to add spaces to the items on the menu page. This can be easily implemented by adding either margin or padding to each of the elements in the menu tab.